Ramda/Extensions/aperture.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  aperture.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 27/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Returns a new list, composed of n-tuples of consecutive elements.
+     If n is greater than the length of the list, an empty list is returned.
+
+     Acts as a transducer if a transformer is given in list position.
+     See also transduce
+
+     - parameter value: must conform to the EmptyDefinable protocol
+
+     - returns: an empty collection type (Array, Dictionary, Set, String)
+
+     */
+
+    public class func aperture(_ number: Int, _ value: [Int]) -> [[Int]] {
```

Comment:
I added aperture in the other PR which has not been merged in yet. Added you as a reviewer. So there will be conflicts. We can get mine in, and then I can resolve conflicts on your branch.

Reply:
I noticed you added aperture as well after I pushed my changes. Thanks for adding me as a reviewer. I agree yours looks cleaner.

Reply:
I removed my aperture implementation and accepted your pull request with the other additions